challenges/platform/troubleshooting.md

```diff
@@ -19,44 +17,46 @@ The interview team and candidate will join a scheduled zoom call for the trouble
 - 15m - Wrap up, Q&A, etc.
 
 ## Areas of Focus
-We are not looking for any specific application knowledge. The test is designed to troubleshoot generic configuration or system faults on any software deployed to a generic Linux host. While this does not require any deployed application knowledge, some familiarity with collecting logs or investigating unfamiliar software is part of the process.
+Our test is designed to cover a range of configuration errors and system faults encountered on a Linux server. Successful candidates will be familiar with collecting logs from and investigating unfamiliar software in order to diagnose underlying system problems. We avoid challenges that require application specific expertise; Instead, we use applications as a jumping off point for diagnosing underlying system issues.
 
 The faults are present in:
 - kernel / systemd / cgroups
-- filesystem or volumes
+- filesystems and volumes
 - networking
 - TLS connectivity
-- Application behaviour
+- Application behavior
```

Comment:
The Americans are taking over with their version of spelling things :)

Reply:
I didn't even think about that -- my vim spellcheck had just flagged this word.  I'll have to add alternative versions :laughing: